pest/tests/calculator.rs

```diff
@@ -187,14 +190,15 @@ fn expression() {
 }
 
 #[test]
-fn prec_climb() {
```

Comment:
I see tests for `PrecClimber` removed and replaced, but `PrecClimber` is still in the codebase. I don't think these should be removed until `PrecClimber` itself is.